src/Controls/src/Core/HandlerImpl/ContentView/ContentView.Impl.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class ContentView : IContentView
+	{
+		object IContentView.Content => Content;
+		IView IContentView.PresentedContent => ((this as IControlTemplated).TemplateRoot as IView) ?? Content;
+
+		protected override void OnApplyTemplate()
```

Comment:
Can you add a test in `Controls.Device.Tests` for this?

Reply:
Done!